thoth/analyzer/__init__.py

```diff
@@ -5,5 +5,7 @@
 from .command import CommandResult
 from .command import run_command
 
+from thoth.common import __version__ as __common__version__
+
 __title__ = 'thoth-analyzer'
-__version__ = "0.1.0"
+__version__ = f"0.1.0+common.{__common__version__}"
```

Comment:
This will not work correctly. We are releasing analyzer as a library - this string is maintained by kebechet and used in setup.py verbatim.

Reply:
why not? its a semver string

Reply:
Kebechet will open an issue complaining this string `` f"0.1.0+common.{__common__version__}"`` not valid semver. It will not be expanded (we do not run code, we parse the version string, so there will be always ``{__common__version__}``).